src/constants/db_keys.py

```diff
@@ -12,7 +12,9 @@
 PRE_PROCESSING = "pre_processing_job_id"
 POST_PROCESSING = "post_processing_job_id"
 STUDENT_JOBS_LEFT = "student_jobs_left"
-INFO = "info"
+RESULTS = "results"
```

Comment:
Could you change the names of the keys to `RESULTS_KEY` It is a bit confusing to see RESULTS = RESULTS

Reply:
if you see the usage in code, I import this file as `db_key` then use these keys as `db_key.RESULT` etc. It seemed repetitive having it like `db_key.RESULT_KEY`.

Should I change how I import these keys then?

Reply:
It's confusing on first read, but It does make a bit more sense.

Reply:
Not changing these for now then